deblur/parallel_deblur.py

```diff
@@ -0,0 +1,71 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015, The Deblur Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+from __future__ import division
+
+from os.path import basename, join, split, splitext
+from os import mkdir
+
+from bfillings.sortmerna_v2 import build_database_sortmerna
+from skbio.parse.sequences import parse_fasta
+
+from qiime.parallel.util import ParallelWrapper
+
+
+class ParallelDeblur(ParallelWrapper):
```

Comment:
I do really think that `multiprocessing` is sufficient. You can see an example [here](https://github.com/biocore/American-Gut/blob/master/americangut/parallel.py) with execution coming from [here](https://github.com/biocore/American-Gut/blob/master/ipynb/primary-processing/09-per_participant_results.md) (see dispatcher call toward the end)